wezterm-gui/src/termwindow/render.rs

```diff
@@ -145,6 +145,281 @@ const PLUS_BUTTON: &[Poly] = &[
+                intensity: BlockAlpha::Full,
+                style: PolyStyle::OutlineThin,
+            },
+        ];
+
+        pub fn sized_poly(poly: &'static [Poly]) -> SizedPoly {
+            let scale = 72.0 / 96.0;
+            let size = Dimension::Points(10. * scale);
+            SizedPoly {
+                poly,
+                width: size,
+                height: size,
+            }
+        }
+    }
+
+    pub(super) mod gnome {
+        use super::*;
+        pub const CLOSE: &[Poly] = &[Poly {
+            path: &[
+                PolyCommand::LineTo(BlockCoord::One, BlockCoord::One),
+                PolyCommand::MoveTo(BlockCoord::One, BlockCoord::Zero),
+                PolyCommand::LineTo(BlockCoord::Zero, BlockCoord::One),
+            ],
+            intensity: BlockAlpha::Full,
+            style: PolyStyle::Outline,
+        }];
+
+        pub const HIDE: &[Poly] = &[Poly {
+            path: &[
+                PolyCommand::MoveTo(BlockCoord::Zero, BlockCoord::Frac(15, 16)),
+                PolyCommand::LineTo(BlockCoord::One, BlockCoord::Frac(15, 16)),
+            ],
+            intensity: BlockAlpha::Full,
+            style: PolyStyle::Outline,
+        }];
+
+        pub const MAXIMIZE: &[Poly] = &[Poly {
+            path: &[
+                PolyCommand::LineTo(BlockCoord::Frac(1, 16), BlockCoord::Frac(15, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(15, 16), BlockCoord::Frac(15, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(15, 16), BlockCoord::Frac(1, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(1, 16), BlockCoord::Frac(1, 16)),
+            ],
+            intensity: BlockAlpha::Full,
+            style: PolyStyle::Outline,
+        }];
+
+        pub const RESTORE: &[Poly] = &[Poly {
+            path: &[
+                PolyCommand::MoveTo(BlockCoord::Frac(3, 16), BlockCoord::Frac(3, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(3, 16), BlockCoord::Frac(13, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(13, 16), BlockCoord::Frac(13, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(13, 16), BlockCoord::Frac(3, 16)),
+                PolyCommand::LineTo(BlockCoord::Frac(3, 16), BlockCoord::Frac(3, 16)),
+            ],
+            intensity: BlockAlpha::Full,
+            style: PolyStyle::Outline,
+        }];
+
+        pub fn sized_poly(poly: &'static [Poly]) -> SizedPoly {
+            let size = Dimension::Pixels(8.);
+            SizedPoly {
+                poly,
+                width: size,
+                height: size,
+            }
+        }
+    }
+}
+
+fn window_button_element(
+    window_button: TabBarItem,
+    is_maximized: bool,
+    font: &Rc<LoadedFont>,
+    metrics: &RenderMetrics,
+    colors: &TabBarColors,
+    style: window::FancyWindowDecorationsStyle,
+) -> Element {
+    use window::FancyWindowDecorationsStyle as Style;
+
+    #[cfg(windows)]
+    let style = Style::Windows;
```

Comment:
Can we remove this? In general, I'd love to avoid as many system-dependent overrides like this in the main body of code as possible.